build-tools/packages/build-cli/src/lib/bump.ts

```diff
@@ -124,6 +135,10 @@ export async function bumpPackageDependencies(
 					newRangeString = depNewRangeOrBumpType;
 				}
 
+				if (isWorkspace === true && preserveWorkspace === true) {
```

Comment:
assuming these are both typed as Booleans, 
```suggestion
				if (isWorkspace && preserveWorkspace) {
```